src/com/videojs/providers/HTTPVideoProvider.as

```diff
@@ -261,7 +261,6 @@ package com.videojs.providers{
         public function load():void{
             _pauseOnStart = true;
             _isPlaying = false;
-            _isPaused = true;
```

Comment:
It's [the default](https://github.com/videojs/video-js-swf/blob/c43b90ab68b08fa9bb0b4bb8290f5b4914576c18/src/com/videojs/providers/HTTPVideoProvider.as#L60)
But we might need to make sure it's paused at the end of a video.